shell/platform/darwin/ios/framework/Source/flutter_main_ios.mm

```diff
@@ -15,7 +15,11 @@ void FlutterMain() {
 
   NSBundle* mainBundle = [NSBundle mainBundle];
 
-  NSString* flutterAssetsPath = [mainBundle pathForResource:@"flutter_assets" ofType:nil];
+  NSString* flutterAssetsPath = [mainBundle objectForInfoDictionaryKey:@"FLTAssetsPath"];
```

Comment:
This should be `NSString *flutterAssetsName = ...`

Then you default `flutterAssetsName` to "flutter_assets"

Then you still need to call `[bundle pathForResource:flutterAssetsName ofType:nil];` to get `flutterAssetsPath`

Reply:
Oops, done.